### PR TITLE
Add beta site banner

### LINKED ISSUE #605: Task: update header to incorporate latest "This is a Beta site..." alert design
## Describe the task
Update header to incorporate latest alert design

## Acceptance Criteria
- [ ] headers for all pages match designs mocked up in designs (happens to currently in recently updated "Breakpoints exploration" section)
- [ ] remove all other alerts from all pages

## Additional context
https://usds.slack.com/archives/C01LA9YLNE9/p1630006337016200

* Add any other context about the task here.
* Or here

### PR DESCRIPTION
**Pull request by vim-usds**
Closes #605 

Product / QA changes:
1. Adds a `beta banner`
2. removes `alerts` from other pages

Tech changes:
- refactors J40Header and BetaBanner to CSS modules using USWDS

### CODE REVIEW

File: client/src/intl/en.json [hunk 1]
```diff
@@ -187,6 +171,14 @@
     "defaultMessage": "Percentile (0-100)",
     "description": "the population of the feature selected"
   },
+  "banner.beta.info": {
+    "defaultMessage": "It is an early, in-progress version of the tool with limited data sets that will be continuously updated.",
```
Comment (switzersc-usds) [inline client/src/intl/en.json:175]: Is this the right message? It just seems weird that it's "It is an early..." as opposed to something like "This is an early..." (What is `it` referring to?)

File: client/src/intl/en.json [hunk 2]
```diff
@@ -187,6 +171,14 @@
...
+    "description": "the main title of the beta banner"
```
Comment (switzersc-usds) [inline client/src/intl/en.json:176]: should this be `main message of...`?

File: client/src/components/AboutCard/AboutCard.test.tsx [hunk 3]
```diff
@@ -4,19 +4,70 @@ import {LocalizedComponent} from '../../test/testHelpers';
...
+          Content body of the action card.
+          </AboutCard>
+        </LocalizedComponent>,
+    );
+    expect(asFragment()).toMatchSnapshot();
+  });
 
-  it('checks if component renders', () => {
+  it('checks if large cards component renders as internal link', () => {
+    const {asFragment} = render(
```
[the remark this answers is not in the document]
Comment (vim-usds) [reply client/src/components/AboutCard/AboutCard.test.tsx:57]: @shelby, 

Originally we had one test, however we weren't testing all the variations of that component. The about card has the following props:

```ts
interface AboutCardProps {
  imgSrc: string;
  header: string;
  size: 'small' | 'large';
  linkText?: string;
  url?: string;
  openUrlNewTab?: boolean;
  className?: string;
  internal?:boolean;
  // body:string | React.ReactNode;
}
```

Each field in the above interface will render the component in a different way. Ideally the test should test all variations. Adding these other variations of the same constant are allowing for increasing coverage on this component.

File: client/src/components/AboutCard/__snapshots__/AboutCard.test.tsx.snap [hunk 4]
```diff
@@ -1,6 +1,126 @@
 // Jest Snapshot v1, https://goo.gl/fbAQLP
 
-exports[`rendering of the AboutCard checks if component renders 1`] = `
+exports[`rendering of the AboutCard checks if large cards component renders 1`] = `
+<DocumentFragment>
```
[the remark this answers is not in the document]
Comment (vim-usds) [reply client/src/components/AboutCard/__snapshots__/AboutCard.test.tsx.snap:4]: @switzersc-usds Awesome catch! The variations mentioned above are being captured by another component. Updating tests and snapshots to reduce redundancy!

**Pull request merged**